go.mod

```diff
@@ -106,6 +105,7 @@ require (
 	github.com/libp2p/go-buffer-pool v0.1.0 // indirect
 	github.com/libp2p/go-cidranger v1.1.0 // indirect
 	github.com/libp2p/go-libp2p-asn-util v0.2.0 // indirect
+	github.com/libp2p/go-libp2p-core v0.20.1 // indirect
```

Comment:
At some point, this will also need to disappear. This is coming from a transitive dependency, so it's fine for now.